src/Integration/WPCOMProxy.php

```diff
@@ -183,6 +183,12 @@ function ( $response, $handler, $request ) {
 						'value' => $this->shipping_time_query->get_all_shipping_times(),
 					];
 
+					$data[] = [
+						'id'    => 'language',
```

Comment:
Wouldn't it be better to add a prefix to this id? I'm concerned that the language is too generic and we might end up overriding this field if WC adds it in the future.

Reply:
Updated here. https://github.com/woocommerce/google-listings-and-ads/pull/2436/commits/719b9a9f08ae9c7b1c46f296b4ff09b90442a2b5